App.js

```diff
@@ -323,8 +304,18 @@ export default class App extends React.Component {
                   component={ChatScreen}
                   options={{headerShown: false}}
                   initialParams={{ id: 1 }} />
-                <Stack.Screen name="Search" component={SearchScreen} options={{headerShown: false}} />
-                <Stack.Screen name="SettingsStack" component={SettingsStackNavigator} options={{headerShown: false}} />
+                <Stack.Group>
+                  <Stack.Screen
+                    name="Search"
+                    component={SearchScreen}
+                    options={{headerShown: false}}
+                  />
+                  <Stack.Screen
+                    name="SettingsStack"
+                    component={SettingsStackNavigator}
+                    options={{headerShown: false}}
+                  />
+                </Stack.Group>
```

Comment:
What does the `Stack.Group` here achieve?

Reply:
At the moment, it only groups logically connected screens ie, all screens in this group are right hand modal. 

Not doing it here, but we could then pass the screen options to the group which is applied to all the screens in the group

Reply:
Oh ok so it just does nothing for now... but we will use it for something?

Reply:
> we will use it for something?

Yes I think we should use the groups to group all right hand side modal stacks/screens, also all the report screens as we can pass options through the group to all the screens